iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ShuntCompensatorAdderImpl.java

```diff
@@ -177,6 +180,17 @@ public ShuntCompensatorAdder setTargetDeadband(double targetDeadband) {
     public ShuntCompensatorImpl add() {
         String id = checkAndGetUniqueId();
         TerminalExt terminal = checkAndGetTerminal();
+        ShuntCompensatorModelWrapper model = null;
```

Comment:
I think this class is redundant with AbstractShuntCompensatorModel
@MioRtia Is it ok if we remove it?

Reply:
Indeed, you can only use `AbstractShuntCompensatorModel` and make the following methods abstract:
```java
ShuntCompensatorModelType getType();

    int getMaximumSectionCount();

    double getB(int sectionCount);

    double getG(int sectionCount);
```